app/controllers/enrolment/EnrolmentProxyController.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.enrolment
+
+import javax.inject.Inject
+
+import config.FrontendAppConfig
+import controllers.actions.{AuthAction, ServiceInfoAction}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import utils.Enrolments
+
+class EnrolmentProxyController @Inject()(
```

Comment:
Does this name still best describe the controller as it only deals with shared workspaces?